src/controllers/scriptController.js

```diff
@@ -73,6 +73,15 @@ const scriptController = {
     // run the script through playData and save the results to the database
     const path = process.env.UPLOADPATH + req.file.filename;
     const scriptData = parseScript(path);
+
+    // if scriptData isn't found, that means the file was invalid and deleted
+    if (!scriptData) {
+      return next({
+        log: `Invalid file, could not identify a title`,
```

Comment:
Add the path of the script for context? Getting this error logged on its own isn't too helpful from a debugging standpoint since we don't know what was invalid about it.